docs/python-sdk/index.mdx

```diff
@@ -25,10 +25,26 @@ source .venv/bin/activate
 
 2. Install the `fused` package:
 
+Install the base package:
 ```bash
 pip install fused
 ```
 
+Or install with optional dependencies:
+```bash
+# For batch processing capabilities
+pip install "fused[batch]"
```

Comment:
I would maybe leave out this one, because it is not entirely clear what this "batch processing" exactly means (it's not that there are dependencies in there that are specifically needed to run an EC2 batch job), and in practice this set of optional packages is almost the same as `fused[all]`